console/atest-ui/src/views/TestCase.vue

```diff
@@ -721,6 +722,22 @@ function formChange() {
   }
 }
 
+function handleSelectChange() {
+  const items = testCaseWithSuite.value.data.request.filename
+  if (items) {
+    testCaseWithSuite.value.data.request.form = [{
+          key: "file",
+          value: items
+        } as Pair]
+
+    const file = fileLists.value.find((e) => e.value === items)
+    if (file) {
+      const content = file.content.split(',')[1]
+      testCaseWithSuite.value.data.request.body = atob(content)
```

Comment:
You can keep it as the base64 encoded content. The below function could decode it on the server side.

https://github.com/LinuxSuRen/api-testing/blob/3a9bde8c06b25de06a58b8f49fe3cbc565a9aedf/pkg/testing/case.go#L186

The decoded content might have many invisible characters. See also

![image](https://github.com/user-attachments/assets/d828cb04-200c-44be-81ca-74e3a48e2526)